src/routes/services/passport-strategies.js

```diff
@@ -0,0 +1,40 @@
+import passport from 'passport';
+import dotenv from 'dotenv';
+import models from '../../db/models';
+
+dotenv.config();
+
+const { User } = models;
+const FacebookStrategy = require('passport-facebook').Strategy;
+
+const credentials = {
+  facebook: {
+    clientID: process.env.FACEBOOK_APP_CLIENT_ID,
+    clientSecret: process.env.FACEBOOK_APP_SECRET,
+    callbackURL: 'http://localhost:3000/api/v1/auth/facebook/callback',
```

Comment:
you should also create an environment variable for this, as this will not work when pushed to Heroku.

Reply:
Yes. I have added the environmental variables to Travis and have asked @NedyUdombat to add it also to the heroku app